pages/linux/eval.md

```diff
@@ -0,0 +1,11 @@
+# eval
+
+> Execute arguments as a single command in the current shell and return its result.
+
+- Call `echo` with the `foo` argument:
+
+`eval "{{echo}}" "{{foo}}"`
```

Comment:
I think `eval {{"echo foo"}}` might be simpler.

Reply:
The whole point of `eval` is to create commands from separate arguments.

Reply:
Practically I don't think it matters much. The point is to be able to construct the string. If the value "foo" comes from a variable, then you can either do `eval "echo" $foo` or `eval "echo $foo"`. Both will give the same result. Conceptually, it might be simpler to imagine the whole command within quotes and that `eval` is evaluating the entire string. Hence my suggestion.

Let me know your thoughts.